hsmd/hsm_client_wire_csv

```diff
@@ -90,3 +91,16 @@ hsm_cupdate_sig_req,,cu,culen*u8
 hsm_cupdate_sig_reply,103
 hsm_cupdate_sig_reply,,culen,u16
 hsm_cupdate_sig_reply,,cu,culen*u8
+
+# Master asks HSM to sign a commitment transaction.
+hsm_sign_commitment_tx,5
+hsm_sign_commitment_tx,,peer_id,struct pubkey
+hsm_sign_commitment_tx,,channel_dbid,u64
+hsm_sign_commitment_tx,,tx,struct bitcoin_tx
```

Comment:
Shouldn't we be generating the tx on the `hsmd` side?

Reply:
Thinking about this some more, we should probably just pass in additional information to allow `hsmd` to sanity check the tx. Same goes for some of the other messages as well.